object-details/design.md

```diff
@@ -0,0 +1,29 @@
+# Object details
+
+
+## Sidecar field
+![](img/sidecar-suggestion.png)
+- We could add a `sidecar` field to the item details to illustrate whether the sidecar is missing or represent
+
+## Minor page structure improvements
```

Comment:
@cshinn I will add another story for you to investigate a design utilizing the PF3 React data grid component rather than the list view which is currently being used.  I think this would also be helpful, what do you think?

Reply:
That sounds good. Is that the component used in OKD right now?

Reply:
👍